docs/make-docs

```diff
@@ -6,6 +6,22 @@
 # [Semantic versioning](https://semver.org/) is used to help the reader identify the significance of changes.
 # Changes are relevant to this script and the support docs.mk GNU Make interface.
 #
+# ## 6.1.0 (2024-04-22)
+#
+# ### Changed
+#
+# - Mount volumes with SELinux labels.
+#
+#   https://docs.docker.com/storage/bind-mounts/#configure-the-selinux-label
+#
+# ## 6.1.0 (2024-04-22)
```

Comment:
(nit) this header probably doesn't need repeating

Reply:
Good catch! I'll fix that upstream when I next make changes to the script to avoid a bunch of PRs just to update the comment mistake